### PR TITLE
Add support for using cert manager to issue TLS certs

### PR DESCRIPTION
**Pull request by hardillb**
## Description

This allows K8s to use cert-manager.io to issue TLS certs for both the core Forge apps and the Instances.

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [x] Documentation has been updated
    - [ ] Upgrade instructions
    - [x] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: helm/flowforge/templates/broker.yaml [hunk 1]
```diff
@@ -153,6 +156,12 @@ spec:
                 name: flowforge-broker
                 port:
                   number: 1884
+  {{- if .Values.ingress.certManagerIssuer }}
+  tls:
+  - hosts:
+    - mqtt.{{ .Values.forge.domain }}
+    secretName: broker-tls
```
Comment (ppawlowski) [inline helm/flowforge/templates/broker.yaml:163]: I suggest renaming secret name to the value of domain name (`mqtt.{{.values.forge.domain}}` in this case).

File: helm/flowforge/templates/service-ingress.yaml [hunk 2]
```diff
@@ -34,3 +37,9 @@ spec:
             name: forge
             port:
               number: 80
+  {{- if .Values.ingress.certManagerIssuer }}
+  tls:
+  - hosts:
+    - {{ $forgeHostname }}
+    secretName: flowforge-tls
```
Comment (ppawlowski) [inline helm/flowforge/templates/service-ingress.yaml:44]: Like with the broker, I suggest renaming the secret name to the value of the domain name (`{{ $forgeHostname }}` in this case).

**Comment by ppawlowski** [comment]
Looks fine, just two small suggestions in the secrets naming convention.

File: helm/flowforge/templates/broker.yaml (hunk 1 above)
Comment (hardillb) [reply helm/flowforge/templates/broker.yaml:163]: Make sense, then it's more obvious what it is

**Review by ppawlowski — APPROVED** [review]

**Pull request merged**